### DISCUSSION TITLE
How to deactivate old session string ?

### DISCUSSION THREAD

**Discussion by tissole** [aahnik/tgcf#88] [Q&A] [answered]
If I generate a session string with the tg-login it will override an old string? I have an old one generated by a different method which might not be secure and I want to delete the old string.

**Comment by aahnik** [comment, marked as answer]
Open your Telegram app's settings and  Settings -> Devices -> Active sessions. There you can find all the sessions that are currently active. You may delete them.

When you generate a session string with [tg-login](https://github.com/aahnik/tg-login) it creates a new session and does not do anything to old ones.

For telegram user accounts you can have 10 sessions active at the same time. For bot accounts only one session active at a time.

**Comment by tissole** [comment]
OK, I got it, deleting all sessions also delete the string. Thanks for the help.
